### PR TITLE
ipyparallel: Add recipe to build 4.1.0.

### PR DESCRIPTION
**Pull request by jakirkham**
Related https://github.com/ContinuumIO/anaconda-recipes/issues/18

### CODE REVIEW

File: ipyparallel/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    - python
+    - ipython_genutils
+    - decorator
+    - pyzmq >=13
+    - ipython >=4
+    - jupyter_client
+    - ipykernel
+    - nose
+    - notebook
+    - tornado >=4
```
Comment (ccordoba12) [inline ipyparallel/meta.yaml]: Please remove the last three dependencies, i.e. nose, notebook and tornado. They are not needed to build or run this package.

**Comment by jakirkham** [comment]
Actually, it was unable to build without those.

**Comment by ccordoba12** [comment]
Please take a look here: https://github.com/ipython/ipyparallel/blob/master/setup.py#L85

The only addition for 4.1 seems to be tornado, not nose or the notebook.

**Comment by jakirkham** [comment]
Oh, that was suppose to be another line note. Sorry, posted on my phone.

Yes, to use everything in `ipyparallel` they are. ( https://github.com/ipython/ipyparallel/issues/73 ) That being said, these are python and some web langs mixed in, which really isn't so much of a burden to install.

**Comment by ccordoba12** [comment]
But the idea of the Jupyter split is to have a very clean installation for each of its packages, not to mix packages that are not needed for runnning or building other components.

So please remove those deps. You can install them separately :-)

**Comment by jakirkham** [comment]
Corrected.

**Comment by ccordoba12** [comment]
Thanks! :-)

**Pull request merged**